core/src/main/scala/com/gu/tableversions/core/InMemoryTableVersions.scala

```diff
@@ -0,0 +1,16 @@
+package com.gu.tableversions.core
+
+/**
+  * Reference implementation of the table version store. Does not persist state.
+  */
+class InMemoryTableVersions[F[_]] extends TableVersions[F] {
```

Comment:
Will this be used outside of tests? If not, I'd probably put it under `test`

Reply:
Thinking about it, it might be useful to have a stub version of this for trying running this code in an EMR cluster against a real metastore + S3... so I might leave it place until we have a real implementation?